src/components/card-container/card-container.js

```diff
@@ -6,8 +6,8 @@ class CardContainer extends React.PureComponent {
   render() {
     const { props } = this;
     const categoryID = props.path.split('#')[1];
-    const cardClasses = classnames('pb12 col col--12', {
-      'col--6-mm': !props.fullWidthCards
+    const cardClasses = classnames('col col--12 mb36', {
+      'col--6-ml': !props.fullWidthCards
```

Comment:
Do you think we need to adjust the width of the images we're using in the cards if we're breaking at `ml` instead of `mm`?

Reply:
@colleenmcginnis from what I can tell from my testing, I don't believe we'll need to adjust the width of the images. Since the images are being rendered as background images with a `background-size: 100%` they should continue to stretch to fit the container

Reply:
Oh! Yes. I was thinkin' if the widest the div can be is going to be wider than it is now, then we might want to bump the `width` we're using in places like [here](https://github.com/mapbox/android-docs/blob/mb-pages/conf/image-config.js#L48). We can wait and see if the images look pixely. 😝

Reply:
Aha! I *think* we should be ok considering most pages that use this component also have a sidebar which takes up a lot of real estate. It's unlikely that an image will stretch beyond 800px, but let's keep an eye on it.